src/main/java/seedu/financialplanner/storage/LoadData.java

```diff
@@ -46,17 +51,82 @@ public static void load(String filePath) throws FinancialPlannerException {
                 default:
                     throw new FinancialPlannerException("Error loading file");
                 }
-
             }
             inputFile.close();
+            deleteFutureCashflows(date);
+            addRecurringCashflows(date);
```

Comment:
can the addrecurringcashflow be done in the income/expense class instead? eg. when making new income/expense object, the recurring is checked in the constructor

Reply:
no